### DISCUSSION TITLE
Suggestions for a few incredibly minor UX improvements on the main menu screen.

### DISCUSSION THREAD

**Discussion by OwenCMYK** [ppy/osu#27102] [Ideas] [answered]
When pressing the space/enter button multiple times on the menu, it takes you down the path of osu>play>solo to get to the beatmap select. This is great from a UX standpoint. In addition, there are some keybinds for certain actions. P for play>solo, E for edit, M for multi, L for playlists, and B/D for browse, Q for quit, and O for options. In addition, any of these keys can be used to press the osu logo. This allows you to spam many of the buttons to get into the mode you want, all except for Multi and Playlists. You cannot mash M to get into multi because M will not press "play" for you, same goes for L with playlists. 

- So firstly, I propose that after the menu first opens, M and L should be able to progress to the play menu, allowing you to press "MMM" at start up in order to get into a multiplayer lobby, or "LLL" for playlists.
- Secondly, I propose that "settings" should be renamed to "options" to match the keybind better, OR the keybind S should be added, which does not currently seem to conflict with anything.
- Thirdy, the keybinds should be able to cancel whatever menu animation is currently playing. Space and Enter already do this, but weirdly P does not despite otherwise doing literally the exact same thing. Animation cancelling functionality should be given to all of these keybinds.

This was inspired by #27080 and is mildly related.

**Comment by peppy** [comment]
> Space and Enter already do this, but weirdly P does not despite otherwise doing literally the exact same thing. Animation cancelling functionality should be given to all of these keybinds.

Can you show what you mean? They do the same thing for me.

**Reply by OwenCMYK** [reply]
If you press PPP too quickly, it won't fully get you to the song select. Whereas if you press space 3 times, it will always go to song select regardless of timing.

**Reply by peppy** [reply]
Right, thanks. Should be fixed via https://github.com/ppy/osu/pull/27393.

**Reply by OwenCMYK** [reply]
Oki doki. Thank you for your hard work peppy!!!

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/27389
